### DISCUSSION TITLE
Customized temp directory on secondary SSD

### DISCUSSION THREAD

**Discussion by igiuseppem** [sepinf-inc/IPED#1548] [General]
Good morning.
As for the subject, I have changed the default temporary directory, in the "LocalConfig.txt", setting a custom directory on a secondary SSD in my workstation.
Just to better understand, let's say that the C: drive is where the operating system is located, instead the D: drive is the SSD where the customized temporary directory is located. Let's say the customized directory name is "temp_dir_custom".
As stated in the description of the "LocalConfig.txt", the "indexTemp" parameter indicates the temporary directory for processing, but after the processing is finished, when I execute the portable case on another workstation, it creates a temporary directory in D: hard drive that belongs to that machine, with the "temp_dir_custom" name, the same I used for the processing. If the workstation where I execute the portable case has not the D: drive, it creates the temporary directory directly on the external storage that has the D: letter assigned.
The question is: when running the portable case, shouldn't the temp directory be created in the OS temp directories?

Kind regards

**Comment by wladimirleite** [comment]
There is an open issue related to that:
https://github.com/sepinf-inc/IPED/issues/1142

**Reply by igiuseppem** [reply]
Ok, sorry.
So, right now, it's better to avoid changing the default temporary directory. Am I right?

**Reply by wladimirleite** [reply]
I just pointed out so you can see what has been discussed. It is still an open question.

What I do in my own cases, once I place it on a media or a network storage, that is going to be accessed by other people (from other machines), I edit the LocalConfig.txt of the case and set temporary location to "default" and remove the path to my hashes database (leave it blank).

**Reply by lfcnassif** [reply]
Hi @igiuseppem, as you are being affected, please read all the discussion @tc-wleite referenced and please vote at the end, I just put a poll there about the possible solutions.

**Reply by igiuseppem** [reply]
Thank you both.
@tc-wleite Yes. I understood that "LocalConfig.txt" was copied to the portable case as it was in the "installation folder", but I was wondering if it could be resetted to default automatically in the portable case.
@lfcnassif Ok, thanks.

**Reply by lfcnassif** [reply]
You're welcome.
